lib/boombox/internal_bin/elixir_stream.ex

```diff
@@ -10,6 +10,10 @@ defmodule Boombox.InternalBin.ElixirStream do
 
   @options_audio_keys [:audio_format, :audio_rate, :audio_channels]
 
+  # the size of the toilet capacity is supposed to handle more or less
+  # the burst of packets from one segment of Live HLS stream
+  @realtimer_toilet_capacity 1000
```

Comment:
I think it should be `more`, not `more or less`, because if it happens to be less, we get a toilet overflow... Are you sure that 1000 is enough for any segment length?

Also, it seems that we now need to set custom toilet capacity wherever we have an `HLS -> manual input` scenario... That sucks, and we should figure out how to solve it